molli/pipeline/nwchem.py

```diff
@@ -49,66 +49,93 @@ class NWChemDriver(DriverBase):
 
     # the nwchem .esp file contains BOTH optimized coordinates and esp charges
     @Job(return_files=("esp.esp",)).prep
-    def optimize_atomic_esp_charges(
+    def optimize_atomic_esp_charges_m(
         self,
         M: Molecule,
         basis: str = "def2-svp",
         functional: str = "b3lyp",
         maxiter: int = 100,
-        memory_total: int = 500, # memory in MB,
-        optimize_geometry: bool = True, # optimize geometry before ESP calc?
-        range: float = 0.2, # nwchem esp params
-        probe: float = 0.1, # nwchem esp params
-        spacing: float = 0.025 # nwchem esp params
-    ):
-        """(Optionally) optimize molecule geometry with NWChem, then calculate atomic esp with NWChem esp module. ESPs are assigned as atom 'nwchem_esp_charge' Atom attributes in the returned Molecule object."""
+        memory_total: int = 500,  # memory in MB,
```

Comment:
Job class has its own memory handling. If you use `self.memory`, you could get the total memory you need here without overriding it in the parameters. It would be great if our interface was consistent with other drivers, such as ORCA.

Reply:
Addressed with 2f84a48